doc/app-METHOD.md

```diff
@@ -0,0 +1,80 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='app.METHOD'>app.METHOD(path, callback [, callback ...])</h3>
+
+Routes an HTTP request, where METHOD is the HTTP method of the request, such as GET,
+PUT, POST, and so on, in lowercase. Thus, the actual methods are `app.get()`,
+`app.post()`, `app.put()`, and so on.  See below for the complete list.
+
+For more information, see the [routing guide](/guide/routing.html).
```

Comment:
> If that is the stance, I don' even see what the point of putting the documentation in this repository is. Just keep them in the website repository, which is the whole point of why they were never in here in the first place. 

Well, that is just my viewpoint as a tech. writer.  @jasnell and others may have a different opinion....as do you I presume.   In my view, having the API docs in markdown in this repo  doesn't make much sense because they are all published on the website.  I'm just doing what I was asked to do, and no one articulated the goal in general (to me anyway).

> To move them in here, means we need to make it work correctly, otherwise there is no purpose, because nothing keeps people from making pull requests to the website repository.

Yes, I guess the rub is what does "work correctly" mean?  To me, it means the doc displays properly in the website, but I think you're proposing that it means that _plus_ displaying properly (at least legibly) in this repo when viewed in GH.  Not unreasonable, but a fair amount of additional work.  And obviously, that goal was not entirely clear to me.

> Ideally, since we need to maintain jsdoc comments for tools like WebStorm, all this is now duplicating all that content anyway. Why don't we just generate the documentation from the jsdoc comments and forget duplicating everything three times?

I've wondered that myself, since I became involved in the project.  IMHO that would be the best solution, but no one has actually proposed it until now.  Having the API doc generated from comments directly in the code is ideal, because it helps to ensure that whenever someone updates the code, the doc is right there in their face.... But there are other "pros and cons" to that approach: for example it's not clear we could make the API docs look as nice as they do now (at least without a lot of work).  

Perhaps we should consider pursuing that approach instead of this?